detox/test/e2e/p-datePicker.js

```diff
@@ -0,0 +1,17 @@
+describe('DatePicker', () => {
+    beforeEach(async () => {
+      await device.reloadReactNative();
+    });
+  
+    beforeEach(async () => {
+      await element(by.text('DatePicker')).tap();
+    });
+    
+    it('check and scroll datePicker', async () => {
+        await expect(element(by.type('UIPickerView'))).toBeVisible();
+        await expect(element(by.type('UIPickerView')).setColumnToValue(1,"6"));
+        await expect(element(by.type('UIPickerView')).setColumnToValue(2,"34"));
```

Comment:
Expect should not be on actions.
You should have a label and see that the correct value is shown there.

Should be something like 
```
await element(by.type('UIPickerView')).setColumnToValue(2,"34");
await expect(element(by.id('time'))).toHaveText('6:34');
```